src/models/user.js

```diff
@@ -29,6 +29,8 @@ export default (sequelize, DataTypes) => {
       // User can belong to a national center through a national center user.
       User.hasMany(models.NationalCenterUser, { foreignKey: 'userId', as: 'nationalCenterUsers' });
       User.belongsToMany(models.NationalCenter, { through: models.NationalCenterUser, foreignKey: 'userId', as: 'nationalCenters' });
+
+      User.hasMany(models.GoalStatusChange, { foreignKey: 'userId', as: 'goalStatusChanges' });
```

Comment:
Just a preference, but I prefer that all association are placed in the file causing the association. The file with the reference, not the model being referenced. It prevents some potential race conditions and keeps each model a little cleaner.